app/src/main/res/values-es/strings.xml

```diff
@@ -57,9 +57,9 @@
     <string name="action_invite">invitación</string>
     <string name="action_drawing">Dibujo</string>
     <string name="action_save_to_gallery">Guardar en la galería</string>
-    <string name="action_select_photo_from_gallery">Select photo from gallery</string> <!-- TODO Add translation -->
-    <string name="action_take_a_photo">Take a photo</string> <!-- TODO Add translation -->
-    <string name="action_reset_avatar">Reset avatar</string> <!-- TODO Add translation -->
+    <string name="action_select_photo_from_gallery">Selecciona una foto de la galería</string>
+    <string name="action_take_a_photo">Toma una foto</string>
```

Comment:
it seems like the standard is to use the infinitive for these, so I think we should change these to "Seleccionar" and "Tomar"